scene/3d/sprite_3d.cpp

```diff
@@ -807,18 +807,13 @@ void AnimatedSprite3D::_draw() {
 		set_base(get_mesh());
 	}
 
-	if (frames.is_null()) {
+	if (frames.is_null() || !frames->has_animation(animation)) {
 		return;
 	}
-
 	if (frame < 0) {
 		return;
 	}
 
-	if (!frames->has_animation(animation)) {
-		return;
-	}
-
```

Comment:
Same here. If you want the null case to return an error separately from the other cases, you need to use `ERR_COND_V()`.

Reply:
See https://github.com/godotengine/godot/pull/65148#discussion_r964949610